firmware/quintuna/BMS/src/hw/hw_cans.c

```diff
@@ -0,0 +1,19 @@
+#include "hw_cans.h"
+#include "main.h"
+
+#include <assert.h>
+
+CanHandle can1 = { .hcan = &hfdcan1 };
+CanHandle can2 = { .hcan = &hfdcan2 };
```

Comment:
how does this work again?  do we have a gpio line saying which to read from? or do we just read from both all the time? do we deinit/reinit how?